contracts/party/PartyGovernance.sol

```diff
@@ -1012,6 +1055,30 @@ abstract contract PartyGovernance is
         voterSnaps.push(snap);
     }
 
+    function _refundGas(uint256 startGas) private {
+        unchecked {
+            // Only refund if the party has ETH.
+            uint256 balance = address(this).balance;
+            if (balance == 0) return;
```

Comment:
shouldn't we also check `balance > refund`?

Reply:
If `balance > refund` it transfers the remaining ETH